docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -500,6 +500,8 @@ Supported modifiers for Format:
 | %D       | Short MM/DD/YY date, equivalent to %m/%d/%y             | 01/02/18   |
 | %e       | day of the month, space-padded ( 1-31)                  | 2          |
 | %F       | short YYYY-MM-DD date, equivalent to %Y-%m-%d           | 2018-01-02 |
+| %g       | two-digit year format, aligned to ISO 8601              | 18         |
```

Comment:
Please clarify `aligned to ISO 8601`.

Reply:
Need information about the difference between "usual" year modificators and these new ones.